operation-update-data-owner.x

```diff
@@ -0,0 +1,37 @@
+%#include "xdr/types.h"
+
+namespace stellar
+{
+
+struct UpdateDataOwnerOp
+{
+    //: ID of the data entry to update an owner
+    uint64 dataID;
+    //: A new owner of the entry
+    AccountID owner;
+    //: Reserved for future extension
+    EmptyExt ext;
+};
+
+enum UpdateDataOwnerResultCode
+{
+    //: An owner of the data was successfully updated
+    SUCCESS = 0,
+    //: `owner` must be in a valid format
+    INVALID_DATA = -1,
```

Comment:
`AccountID` type restricts the range of values that may be used in this field so this error code is not needed